strings/snake_case_to_camel_pascal_case.py

```diff
@@ -0,0 +1,54 @@
+def snake_to_camel_case(input: str, use_pascal: bool = False) -> str:
```

Comment:
Also with capitalize(), but since it is for others to learn, I kept it this way, like the one for split.

Didn’t want to just use the method that already does almost all the job.